.github/workflows/release-pr.yml

```diff
@@ -84,28 +100,52 @@ jobs:
           # 'Stable tag' header in readme.txt;
           sed -i "s/^Stable tag: .*$/Stable tag: $RELEASE_VERSION/" readme.txt
 
-          # Duplicate the generated changelog into the Changelog section from the readme.txt 
+          # Duplicate the generated changelog into the Changelog section from the readme.txt
           sed -ri "s|(== Changelog ==)|\1\n\n= $RELEASE_VERSION - $RELEASE_DATE =\n$CHANGELOG|" readme.txt
-      
+
       - name: "Commit and push changes"
         env:
           RELEASE_VERSION: ${{ steps.create_branch.outputs.trimmed-version }}
           BRANCH_NAME: ${{ steps.create_branch.outputs.branch-name }}
         run: |
           npm install # To update package-lock.json with the new version of the plugin
           git config user.name "github-actions[bot]"
-          # We could consider fetching the bot's ID through an API call to be future-proof. Hardcoded for now. 
+          # We could consider fetching the bot's ID through an API call to be future-proof. Hardcoded for now.
           # See https://github.com/Automattic/woocommerce-payments/pull/5200#discussion_r1034560144
           git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
           git commit -am "Update version and add changelog entries for release $RELEASE_VERSION"
           git push --set-upstream origin $BRANCH_NAME
 
       - name: "Create a PR to trunk"
+        id: create-pr-to-trunk
         env:
           GITHUB_TOKEN: ${{ secrets.BOTWOO_TOKEN }}
           BRANCH_NAME: ${{ steps.create_branch.outputs.branch-name }}
           CHANGELOG: ${{ steps.generate_changelog.outputs.changelog }}
           RELEASE_VERSION: ${{ steps.create_branch.outputs.trimmed-version }}
         run: |
           PR_BODY=$(echo -e ":warning: Please do not merge the PR from the GitHub interface. :warning:\n\n Instead, you can use the following command:\n\`\`\`\n git checkout trunk && git pull \n git merge --no-ff $BRANCH_NAME -m 'Merge $BRANCH_NAME into trunk' \n git push origin trunk \n\`\`\` \n Changelog: \n\`\`\`\n${CHANGELOG}\n\`\`\`")
-          gh pr create --title "Release branch for $RELEASE_VERSION" --body="$PR_BODY" --base="trunk"
+          PR_URL=$(gh pr create --title "Release branch for $RELEASE_VERSION" --body="$PR_BODY" --base="trunk")
+          PR_ID=${PR_URL##*/}
+          if [[ $PR_ID =~ ^[0-9]+$ ]]; then
+            echo "Release PR ID: $PR_ID"
```

Comment:
This `echo` can be removed? I assume it was there during the testing

Reply:
Fixed in https://github.com/Automattic/woocommerce-payments/pull/5586/commits/294f03bd554f2a0fdb88753af87032b0fb64cdc0